modules/recon/profiles-contacts/bing_linkedin_contacts.py

```diff
@@ -1,14 +1,15 @@
 from recon.core.module import BaseModule
 from recon.mixins.search import BingAPIMixin
 from recon.utils.parsers import parse_name
+import re
 
 
 class Module(BaseModule, BingAPIMixin):
 
     meta = {
         "name": "Bing LinkedIn Profile Contact Harvester",
         "author": "Cam Barts (@cam-barts)",
-        "version": "1.1",
+        "version": "1.2",
```

Comment:
Looks like you bumped the version, but didn't run the `index.py` script to change the `modules.yml`. Would you mind doing that?

Reply:
You bet